erpnext/selling/page/point_of_sale/pos_past_order_summary.js

```diff
@@ -108,6 +114,25 @@ erpnext.PointOfSale.PastOrderSummary = class {
 				)}</div>`;
 			}
 		}
+
+		async function get_returned_qty() {
+			const r = await frappe.call({
+				method: "erpnext.controllers.sales_and_purchase_return.get_pos_inv_item_returned_qty",
+				args: {
+					invoice: doc.name,
+					customer: doc.customer,
+					item_name: item_data.name,
```

Comment:
The variable `item_name` is misleading. It's not the item's name, it's actually `item-row-name`.